domino/formula/pom.xml

```diff
@@ -31,6 +31,12 @@
 				<extensions>true</extensions>
 				<configuration>
 					<singleton>true</singleton>
+					<manifestLocation>META-INF</manifestLocation>
+					<source>${compiler}</source>
+					<target>${compiler}</target>
```

Comment:
Maybe we can move the $compiler configuration up to the domino/root and put the default configuration of the maven-bundle-plugin  in a <pluginManagement> section